templates/robotics/ros-esm.html

```diff
@@ -44,251 +39,400 @@ <h4>
-    <div class="col-8">
-      <h3>
-        Reduced attack surface
-      </h3>
-      <p>
-        Running un-patched and unmaintained versions of ROS exposes both your robot and your customers to serious risk.
-      </p>
-      <p>
-        ROS ESM offers backported security fixes to your ROS environment for a long-term supported system beyond EOL.
+    <hr class="is-fixed-width">
+    <div class="p-logo-section">
+      <p class="p-logo-section__title">
+        A few of the 23,000 packages included in Ubuntu Universe:
       </p>
     </div>
-    <div class="col-4 u-align--center u-vertically-center u-hide--medium u-hide--small">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/aa1dc625-10+Year+shield+trio_AW.svg",
-        alt="",
-        width="263",
-        height="150",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+    <div class="p-logo-section__items u-align--left">
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/8449ec67-144px%20Python%20logo.svg",
+          alt="Python",
+          width="144",
+          height="144",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-logo-section__logo"}
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        <img src="https://assets.ubuntu.com/v1/ad800509-144px%20Boost%20logo.svg" alt="Boost C++ libraries" class="p-logo-section__logo">
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/98d2bf31-144px%20QT%20logo.svg",
+          alt="QT",
+          width="144",
+          height="144",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-logo-section__logo"}
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/74ba014d-144px%20FFmpeg%20logo.svg",
+          alt="FFmpeg",
+          width="144",
+          height="144",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-logo-section__logo"}
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        <img src="https://assets.ubuntu.com/v1/c1c372ec-144px%20pcl%20logo.svg" alt="PCL" class="p-logo-section__logo">
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/2bc804f3-144px%20Cython%20logo.svg",
+          alt="Cython",
+          width="144",
+          height="144",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-logo-section__logo"}
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/8be1ceef-144px%20GTK%20logo.svg",
+          alt="GTK",
+          width="144",
+          height="144",
```

Comment:
```suggestion
          height="288",
```

Reply:
@anthonydillon if the logos must be raster images, this is one way to solve the problem, but of course at the cost of slower loading.